src/pathfinding_service/model/token_network.py

```diff
@@ -10,7 +10,7 @@
     DIVERSITY_PEN_DEFAULT,
     MAX_PATHS_PER_REQUEST,
 )
-from pathfinding_service.model import ChannelView
+from pathfinding_service.model.channel_view import ChannelView
```

Comment:
How could this work before?

Reply:
`pathfinding_service/model/__init__.py` does an `import *` from it's submodules. The main reason I changed the import is that it caused a circular import, before. It did work, but I prefer to avoid circular imports.